src/components/grid/col.js

```diff
@@ -34,6 +34,15 @@ const Col = styled.div`
     `)}
   `}
 
+  ${p => p.order && css`
```

Comment:
here, I suggest these changes to work with numerical values ​​including negative numbers
```suggestion
  ${p => p.order && css`
    ${DIMENSIONS.map(d =>
    config(p).breakpoints[d] && config(p).media[d]`
    ${typeof p.order === 'object'
    ? p.order[d] !== undefined && `order: ${p.order[d]}`
    : `order: ${p.order};`};
    `)}
  `}
```

Reply:
Oh! Yes, you right! I will fix